neon_messagebus_mq_connector/controller.py

```diff
@@ -98,27 +106,31 @@ def bus(self) -> MessageBusClient:
             self.connect_bus()
         return self._bus
 
-    def handle_neon_message(self, message: Message,
-                            routing_key: str = None):
+    def handle_neon_message(self, message: Message):
         """
-        Handles responses from Neon Core
+            Handles responses from Neon Core
 
-        :param message: Received Message object
-        :param routing_key: Queue to post response to
+            :param message: Received Message object
         """
 
         if not message.data:
             message.data['msg'] = 'Failed to get response from Neon'
-
-        body = {'msg_type': message.msg_type,
-                'data': message.data, 'context': message.context}
+        message.context.setdefault('klat', {})
```

Comment:
As discussed during the meeting, consider refactoring `klat` to `klat_data` since that's what is checked in the templates and in the audio module for emitting audio responses back here (klat.response)